RMSD including initial alignment by principal moments of inertia

aarongarrison opened the pull request:
Added in functions to compute moments of inertia and principal moments of inertia for mol3D objects. Also added in an RMSD function that performs `rigorous_rmsd` logic (reorders the atoms based on pairwise distances and then rotates the molecules to align) after aligning the molecules about their principal moments of inertia, which should be more robust to errors in improper assignments.

dbkchu commented:
Also could update the test case that shows issues with rigorous_rmsd https://github.com/hjkgrp/molSimplify/blob/d51bd5015d1423e4e9d8da3d29d8f8d039358473/tests/test_rmsd.py#L36

jwtoney commented:
Looks good to me as well, agree with Daniel's suggestion for test cases

aarongarrison commented:
Added in a new test case with the trigonal bipyramidal structure mentioned in https://github.com/hjkgrp/molSimplify/issues/127, and modified the methane rotation example to show that it is handled by `align_rmsd`.
